### DISCUSSION TITLE
Support for abstract unix domain socket

### DISCUSSION THREAD

**Discussion by lakpic-personal** [nodejs/help#4762] [Q&A] [answered]
Hello,
Can Node server listen to an abstract unix domain socket (e.g @socket-name which internally maps to first character being a null terminator).  More info on abstract namespaces from [linux man page](https://man7.org/linux/man-pages/man7/unix.7.html) I've seen it working with path name socket, but not sure about the abstract unix domain socket.

**Comment by bnoordhuis** [comment, marked as answer]
Short answer: no, not at the moment.

Longer answer: it's blocked on libuv support. I've opened an issue: https://github.com/libuv/libuv/issues/4028

**Comment by ggoodman** [comment]
Hi @bnoordhuis, now that the `libuv` support seems to have landed, do you think that this is something we might be able to pick up in Node.js?

**Reply by bnoordhuis** [reply]
Yes. Node needs to switch to the new API. I don't plan on working on that anytime soon (rocks to climb) but you're welcome to take it up.
